octopuslite/reader.py

```diff
@@ -104,43 +110,55 @@ def __getitem__(self, channel_name: Union[str, Channels]):
 
         return self._lazy_arrays[channel_name]
 
-
     def files(self, channel_name: str) -> list:
         return self._files[Channels[channel_name.upper()]]
 
     def _load_and_process(self, fn: str) -> np.ndarray:
         """Load and crop the image."""
         image = io.imread(fn)
 
-        t = int(parse_filename(fn)['time'])
-        image = self._transformer(image, t)
+        if self._transformer is not None:
```

Comment:
What happens if `self._transformer` is None?

Reply:
it skips over the transformation implementation

Reply:
wait, no it doesnt... need to sort this out before merging

Reply:
sorted that out now